src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudServiceTest.kt

```diff
@@ -265,12 +385,28 @@ class CosmasGoogleCloudServiceTest {
                 .newBuilder()
                 .setProjectId(projectId)
                 .build()
-
         return Pair(commitRecorder, commitRequest)
     }
 
     private fun commit(projectId: String = "0") {
         val (commitRecorder, commitRequest) = getStreamRecorderAndRequestForCommitVersions(projectId)
         this.service.commitVersion(commitRequest, commitRecorder)
     }
+
+    private fun addPatchToService(text: String, user: String, fileId: String, time: Long, projectId: String) {
```

Comment:
Please keep the names consistent. If you use `id` suffix to indicate that it is identifier, use it always. Otherwise your reader may think that `user` is not actually an id (name maybe?)

Reply:
ok